openmetadata-service/src/main/java/org/openmetadata/service/apps/bundles/slack/isteners/commands/CommandListeners.java

```diff
@@ -0,0 +1,17 @@
+package org.openmetadata.service.apps.bundles.slack.isteners.commands;
+
+import com.slack.api.bolt.App;
+import org.openmetadata.service.apps.bundles.slack.isteners.ListenerProvider;
+import org.openmetadata.service.formatter.decorators.SlackMessageDecorator;
+
+public class CommandListeners implements ListenerProvider {
+  SlackMessageDecorator decorator = new SlackMessageDecorator();
+
+  @Override
+  public void register(App app) {
+    app.command("/search-glossary", new GlossarySearchCommand(decorator));
```

Comment:
any reason we need to do this. we need one /search not for every and return all the matching assets types.
users can 
/search 'text' which search across all the entities
or /search entityType 'text' which will search with-in
once an asset found they can request more details such as 
/asset fqn or id

Reply:
We can do that. I will update search commands. Noted.